include/openthread/platform/ble.h

```diff
@@ -281,6 +315,21 @@ otError otPlatBleGattServerIndicate(otInstance *aInstance, uint16_t aHandle, con
  */
 extern void otPlatBleGattServerOnWriteRequest(otInstance *aInstance, uint16_t aHandle, const otBleRadioPacket *aPacket);
 
+/**
+ * Function to retrieve from platform BLE link capabilities.
+ *
+ * @param[in] aInstance             The OpenThread instance structure.
+ *
+ */
+otBleLinkCapabilities otPlatGetBleLinkCapabilities(otInstance *aInstance);
```

Comment:
Better to avoid returning `struct` type?
Can consider  passing a pointer to a `otPlatGetBleLinkCapabilities` to this function for it to populate?

Reply:
fixed.